elastic/logs/operations/esql.json

```diff
@@ -1,66 +1,38 @@
-      "path": "/_query",
-      "body": {
-        "query": "FROM logs-* | STATS count=count(*) BY agent.version | SORT count DESC | LIMIT 20"
-      }
+      "operation-type": "esql",
+      "query": "FROM logs-* | STATS count=count(*) BY agent.version | SORT count DESC | LIMIT 20"
     },
     {
       "name": "esql_basic_count_group_2",
-      "operation-type": "raw-request",
-      "method": "POST",
-      "path": "/_query",
-      "body": {
-        "query": "FROM logs-* | STATS count=count(*) BY agent.version, agent.type | SORT count DESC | LIMIT 20"
-      }
+      "operation-type": "esql",
+      "query": "FROM logs-* | STATS count=count(*) BY agent.version, agent.type | SORT count DESC | LIMIT 20"
     },
     {
       "name": "esql_basic_count_group_3",
-      "operation-type": "raw-request",
-      "method": "POST",
-      "path": "/_query",
-      "body": {
-        "query": "FROM logs-* | STATS count=count(*) BY agent.version, agent.type, agent.hostname | SORT count DESC | LIMIT 20"
-      }
+      "operation-type": "esql",
+      "query": "FROM logs-* | STATS count=count(*) BY agent.version, agent.type, agent.hostname | SORT count DESC | LIMIT 20"
     },
     {
       "name": "esql_basic_count_group_4",
-      "operation-type": "raw-request",
-      "method": "POST",
-      "path": "/_query",
-      "body": {
-        "query": "FROM logs-* | STATS count=count(*) BY agent.version, agent.type, agent.hostname, agent.id | SORT count DESC | LIMIT 20"
-      }
+      "operation-type": "esql",
+      "query": "FROM logs-* | STATS count=count(*) BY agent.version, agent.type, agent.hostname, agent.id | SORT count DESC | LIMIT 20"
     },
     {
       "name": "esql_time_range_and_date_histogram_two_groups_pre_filter",
       "description": "Based on observability queries for average CPU over date histogram",
-      "operation-type": "raw-request",
-      "method": "POST",
-      "path": "/_query",
-      "body": {
-        "query": "FROM logs-* | EVAL start_time = DATE_PARSE(\"yyyy-MM-dd\",\"2020-01-01\"), end_time = DATE_PARSE(\"yyyy-MM-dd\",\"2020-01-02\") | WHERE @timestamp >= start_time AND @timestamp <= end_time AND http.response.body.bytes IS NOT NULL | EVAL bucket = DATE_TRUNC(1 hour, @timestamp) | STATS avg=AVG(http.response.body.bytes), min=MIN(http.response.body.bytes), max=MAX(http.response.body.bytes) BY data_stream.dataset, bucket | KEEP data_stream.dataset, bucket, min, avg, max"
-      }
+      "operation-type": "esql",
+      "query": "FROM logs-* | EVAL start_time = DATE_PARSE(\"yyyy-MM-dd\",\"{{ bulk_start_date }}\"), end_time = DATE_PARSE(\"yyyy-MM-dd\",\"{{ bulk_end_date }}\") | WHERE @timestamp >= start_time AND @timestamp <= end_time AND http.response.body.bytes IS NOT NULL | EVAL bucket = DATE_TRUNC(1 hour, @timestamp) | STATS avg=AVG(http.response.body.bytes), min=MIN(http.response.body.bytes), max=MAX(http.response.body.bytes) BY data_stream.dataset, bucket | KEEP data_stream.dataset, bucket, min, avg, max"
```

Comment:
As we discussed, this may become a problem if the range of the `bulk_start_date` and `bulk_end_date` is something like many weeks, or months, as we're hardcoded to try and return 1h buckets.

Given the substantial resources it requires to operate this track it is my experience that this track is usually only ran with a maximum of ~7 days range or so, so perhaps it's not a big deal.